startup/21-cameras.py

```diff
@@ -0,0 +1,107 @@
+print(f'Loading {__file__}...')
+
+
+from ophyd.areadetector import (AreaDetector, ImagePlugin,
+                                TIFFPlugin, StatsPlugin, 
+                                ROIPlugin, TransformPlugin,
+                                OverlayPlugin, ProcessPlugin)
+from ophyd.areadetector.filestore_mixins import (FileStoreIterativeWrite,
+                                                 FileStoreTIFF)
+from ophyd.areadetector.trigger_mixins import SingleTrigger
+from ophyd.areadetector.cam import AreaDetectorCam
+from ophyd.device import Component as C
+
+
+### Triggers for HFM/BPM1/HF VLM cameras
+hfm_cam = EpicsSignal('XF:05IDA-BI:1{FS:1-Cam:1}Acquire_RBV',
+                        write_pv='XF:05IDA-BI:1{FS:1-Cam:1}Acquire',
+                        name='hfm_cam_trigger')
+hfm_tot1 = EpicsSignal('XF:05IDA-BI:1{FS:1-Cam:1}Stats1:Total_RBV',
+                        name='hfm_tot1')
+bpm1_cam = EpicsSignal('XF:05IDA-BI:1{BPM:1-Cam:1}Acquire_RBV',
+                        write_pv='XF:05IDA-BI:1{Mir:1-Cam:1}Acquire',
+                        name='hfm_cam_trigger')
+bpm1_tot1 = EpicsSignal('XF:05IDA-BI:1{BPM:1-Cam:1}Stats1:Total_RBV',
+                         name='bpm1_tot1')
+hfvlm_cam = EpicsSignal('XF:05IDD-BI:1{Mscp:1-Cam:1}cam1:Acquire',
+                        write_pv='XF:05IDD-BI:1{Mscp:1-Cam:1}cam1:Acquire',
+                        name='hfvlm_cam_trigger')
+hfvlm_cam_tiff = EpicsSignal('XF:05IDD-BI:1{Mscp:1-Cam:1}TIFF1:Capture',
+                        write_pv='XF:05IDD-BI:1{Mscp:1-Cam:1}TIFF1:Capture',
+                        name='hfvlm_cam_tiff')
```

Comment:
Those objects defined above are not used anywhere in the profile, according to @tacaswell.